modules/led.sh

```diff
@@ -100,7 +100,18 @@ function led {
       echo "Green LED: on 2 sec; off 0.5 sec"
       echo "Red LED: on 2 sec; off 0.5 sec"
       eid > "$LOGFILE"
-      ;;
+      ;;      
+    wazazi)
+      checkroot
+      echo "leds are set to wazazi mode."
+      echo "Look at your RPi leds, both leds will be in this pattern..."
+      echo "Both LED: 1 sec off"
+      echo "Green LED: flash 6 times, on 2 sec"
```

Comment:
Not too sure about this pattern description, the LED lights seem to flash 6 times alternating between green and red